libcontainer/configs/config.go

```diff
@@ -246,6 +239,109 @@ type Capabilities struct {
 	Ambient []string
 }
 
+//
+// FsEntry type.
+//
+
+type FsEntryKind uint32
+
+const (
+	InvalidFsKind FsEntryKind = iota
+	FileFsKind
+	DirFsKind
+	SoftlinkFsKind
+)
+
+type FsEntry struct {
+	Kind FsEntryKind
+	Path string
+	Mode os.FileMode
+	Dst  string
+}
+
+func NewFsEntry(path, dst string, mode os.FileMode, kind FsEntryKind) *FsEntry {
+
+	entry := &FsEntry{
+		Kind: kind,
+		Path: path,
+		Mode: mode,
+		Dst:  dst,
+	}
+
+	return entry
+}
+
+func (e *FsEntry) Create() error {
```

Comment:
Suggestion: Create() -> Add()   ("create" sounds similar to newFsEntry). Add is clearer and gels with "remove" better too.

Reply:
Ok. Done.